### ISSUE TITLE
Small drives are shown in red even when there is plenty of space

### ISSUE THREAD

**Issue by metov** [muesli/duf#136]
When I create a 1 GB partition on my drive, and write 100 MB of data to it, `duf` highlights it in red. I assume this means low space and indicates that "only" 900 MB free space is left. However, a partition being 10% full hardly seems like something to be alarmed over.

I suppose it is ultimately subjective whether people would care more about absolute or relative free space. Sounds like `duf` is going with the latter. Is there any possibility of focusing on relative space instead?

* In my use case, the partition in question is `/boot` and hence absolutely nothing is wrong.
* Using relative thresholds is self-updating, and obviates the need to perpetually increase the threshold of what counts as "low" as hard drives get bigger and user expectations change.

At the very least, it would be nice to have an option `--color {relative|absolute}` to control this behavior.

**Comment by Justinzobel** [comment]
This is an issue for me also as it shows boot and efi partitions in red even though there's plenty of space:
![image](https://user-images.githubusercontent.com/1370117/149718923-8d65bc41-0e97-4a20-b785-b5dafc20f2f6.png)

**Comment by muesli** [comment]
This kind of works as intended. You'll still get to see a green value for the usage in percent. As I see it, unifying the color coding for both columns would only remove information. I do understand that everyone has different needs however, so maybe `duf`'s recently added `--avail-threshold` and `--usage-threshold` flags help remedy this a bit?

```bash
# Availability threshold, for example in gigabytes
$ duf --avail-threshold="10G,1G"

# Usage-threshold in percent between 0.0 and 1.0
$ duf --usage-threshold="0.5,0.9"
```

**Comment by Justinzobel** [comment]
The problem with the thresholds is they are global I imagine, against all disks.

I don't want it to tell me when only 559.7M is available on my root partition, which could easily fill up in moments with caches and regular activities.

**Comment by metov** [comment]
@muesli Thank you for the response! Those options resolve my issue, so feel free to close.

**Comment by esran** [comment]
Yeah, the problem is that the colour difference really draws the eye. Is it actually useful to be see that there's only a 'small' amount available on a drive that is at 5% use? I guess it might help identify a poorly configured device, but is that really beyond the scope of the tool? My immediate thought was to maybe allow avail-threshold to specified as a fraction (like use) but then I realised I'd only ever set it the same as use anyway.
